### PR TITLE
Remove cybershake cfg

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

**Comment by jpa198** [comment]
Waiting on PR #162 to go through

**Comment by jpa198** [comment]
Passed end2end test

**Comment by jasonmotha** [comment]
Changes requested as python parameters to install.

File: scripts/cybershake/install_cybershake_fault.py [hunk 1]
```diff
@@ -112,59 +105,60 @@ def main():
             print(message)
             with open(error_log, "a") as error_fp:
                 error_fp.write(message)
-            sys.exit()
```
Comment (jasonmotha) [inline scripts/cybershake/install_cybershake_fault.py]: Raise exception?

File: scripts/hf_sim.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def random_seed():
 if is_master:
     logger.debug("=" * 50)
     # random seed
-    seed_file = os.path.join(os.path.dirname(args.out_file), "SEED")
+    seed_file = os.path.join(os.path.dirname(args.out_file), constants.RootParams.seed.value)
```
Comment (jasonmotha) [inline scripts/hf_sim.py]: Seems to be a confusing variable name?

File: testing/test_common_set_up.py [hunk 3]
```diff
@@ -41,4 +41,4 @@ def get_bench_output(root_path, func_name):
 def set_up(request):
     data_locations = testing.test_set_up(REALISATIONS)
     yield list(zip(data_locations, [rel[0] for rel in REALISATIONS]))
-    testing.test_tear_down(data_locations)
+    #testing.test_tear_down(data_locations)
```
Comment (jasonmotha) [inline testing/test_common_set_up.py]: meant to still be commented?

File: scripts/cybershake/install_cybershake.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        "fault_selection_list", type=str, default=None, help="The fault selection file"
+    )
+    parser.add_argument(
+        "--seed", type=str, default=0, help="The seed to be used for HF simulations. Default is to request a random seed."
+    )
+
+    args = parser.parse_args()
+
+    faults = {}
+    with open(args.fault_selection_file) as fault_file:
```
Comment (claudio525) [inline scripts/cybershake/install_cybershake.py]: This argument doesn't exists, I'm guessing this should be "fault_selection_file"?

File: scripts/cybershake/install_cybershake.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        "path_cybershake",
+        type=str,
+        default=None,
+        help="the path to the root of a specific version cybershake.",
+    )
+    parser.add_argument(
+        "version", type=str, default="16.1", help="Please specify GMSim version"
+    )
+    parser.add_argument(
+        "fault_selection_list", type=str, default=None, help="The fault selection file"
```
Comment (claudio525) [inline scripts/cybershake/install_cybershake.py]: Why give it a default value? Isn't this a required argument?

File: scripts/cybershake/install_cybershake.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    with open(args.fault_selection_file) as fault_file:
+        for line in fault_file.readlines():
+            fault, count, *_ = line.split(" ")
+            count = int(count[:-1])
+            faults.update({fault: count})
+
+    for fault, count in faults.items():
+        try:
+            install_fault(fault, count, args.path_cybershake, args.version, args.seed)
+        except RuntimeError as e:
```
Comment (claudio525) [inline scripts/cybershake/install_cybershake.py]: Do we want this to be broader (i.e. Exception), or are we okay with it stopping installation if there is an unexpected error? Just pointing this out as it is different than how the bash script handled errors

File: scripts/cybershake/install_cybershake_fault.py [hunk 7]
```diff
@@ -34,19 +34,22 @@ def main():
     parser.add_argument(
         "--n_rel", type=int, default=None, help="the number of realisations expected"
     )
+    parser.add_argument(
+        "--seed", type=int, default=None, help="The seed to be used for HF simulations"
```
Comment (claudio525) [inline scripts/cybershake/install_cybershake_fault.py]: Don't we want the default value to be 0?

**Pull request merged**